assets/aws/files/bin/teleport-generate-config

```diff
@@ -20,6 +20,10 @@ LOCAL_HOSTNAME=${LOCAL_HOSTNAME//./-}
 # Source variables from user-data
 source /etc/teleport.d/conf
 
+# Create /var/lib/teleport if it doesn't exist and fix permissions appropriately
+if [ ! -d /var/lib/teleport ]; then
```

Comment:
More of a nit, but isn't this check basically redundant since you're running `mkdir -p` that's kind of like an "upsert" for a directory?

Reply:
Yes, it's pretty redundant. I was keen not to alter the current behaviour of the script at all if the directory does exist, so figured I'd only run the command if it doesn't.